src/tz_macos.rs

```diff
@@ -70,7 +82,7 @@ impl<T> Drop for Dropping<T> {
 
 impl<T> Dropping<T> {
     #[inline]
-    unsafe fn new(v: *const T) -> Option<Self> {
+    fn new(v: *const T) -> Option<Self> {
```

Comment:
This method must remain `unsafe`. You can use only safe functions to construct an invalid pointer.

Reply:
Although I can see your argument that `Dropping::new` could be needed to be marked unsafe because the type will deterrence the pointer on drop.